financial-connections/src/main/java/com/stripe/android/financialconnections/features/manualentry/ManualEntryScreen.kt

```diff
@@ -96,97 +100,140 @@ private fun ManualEntryContent(
             )
         }
     ) {
+        when (payload) {
+            is Loading, Uninitialized -> LoadingContent()
+            is Fail -> ErrorContent(
+                error = payload.error,
+                onSelectAnotherBank = {},
+                onEnterDetailsManually = {},
+                onCloseFromErrorClick = {}
+            )
+
+            is Success -> when (payload().customManualEntry) {
+                true -> LoadingContent()
```

Comment:
In the case of custom manual entry, we still get to the manual entry screen, where the session is completed and the flow finalized automatically. While this happens, we want to still show the loading view. 

We handle this logic within the manual entry screen because there are several entry points to it, so this way the logic is centralized.